qa/rpc-tests/wallet_nullifiers.py

```diff
@@ -117,7 +118,7 @@ def run_test (self):
 
         # check zaddr balance
         zsendmany3notevalue = Decimal('1.0')
-        zaddrremaining2 = zaddrremaining - zsendmany3notevalue - zsendmanyfee
+        zaddrremaining2 = zaddrremaining - zsendmany3notevalue - DEFAULT_FEE
```

Comment:
It would probably be a good idea to entirely delete `DEFAULT_FEE` from the tests as part of @daira's changes, perhaps retaining a `LEGACY_DEFAULT_FEE` for block construction tests where we want to ensure that the free actions are handled correctly.

Reply:
This changed because `zsendmanyfee` was previously used both where the call to `z_sendmany` omitted the fee and where it explicitly passed `DEFAULT_FEE`. Now those have two different values, so I kept `zsendmanyfee` for the conventional fee and changed to `DEFAULT_FEE` for the explicit ones.

And I agree – DEFAULT_FEE should be removed. Something like `LEGACY_DEFAULT_FEE` could be local to a single test where we’re explicitly checking behavior of pre-317 wallets. And fixed fees in other places can just be arbitrary (like the many 0-fee tests we have now).

Reply:
Oh, right! I also ran into this in https://github.com/zcash/zcash/pull/6153